modules/builds-configmaps-use-case-consuming-in-volumes.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+//* builds/builds-configmaps.adoc
+
+[id="builds-configmaps-use-case-consuming-in-volumes_{context}"]
+= Consuming ConfigMaps in volumes
+
+A ConfigMap can also be consumed in volumes. Returning again to the following example ConfigMap:
```

Comment:
Echoing my previous comment regarding ' Injecting content into a volume by using ConfigMaps' here and for other references in this module.